test/connect.spec.js

```diff
@@ -36,22 +40,22 @@ describe('connect()', () => {
     })
   })
 
-  it('ERR - channelId not found', () => {
+  it.skip('ERR - channelId not found', () => {
```

Comment:
Do you know that these tests are skipped?

Reply:
Yes, I told you :)

Reply:
I forgot. :)